### DISCUSSION TITLE
Using texture Objects

### DISCUSSION THREAD

**Discussion by kalosu** [inducer/pycuda#453] [Q&A]
Hi there pycuda community,

I was wondering if there is any possibility to integrate support for texture objects in pycuda.

As far as I understand it, in pycuda it is possible to set a texture reference via ```mod.get_texref```

However, texteture references are depreciated from the newest CUDA toolkit versions and they only support texture objects.

Is it possible to bring support for these into pycuda?

**Comment by inducer** [comment]
Dupe of #419.
